src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAOQuerySupport.scala

```diff
@@ -68,13 +74,14 @@ trait ElasticSearchDAOQuerySupport extends ElasticSearchDAOSupport {
       case Some(searchTerm) if searchTerm.trim == "" => matchAllQuery
       case Some(searchTerm) =>
         val fieldSearch = if (phrase) {
-          matchPhraseQuery(searchField, searchTerm).minimumShouldMatch("2<67%")
+          matchPhraseQuery(searchField, searchTerm)
+            //.minimumShouldMatch("2<67%")
```

Comment:
Great question. All our test cases are passing - but I also have this on my list for follow-up, in the optimization phase once we get through compatibility.